packages/contracts-bedrock/scripts/DeployOPChain.s.sol

```diff
@@ -135,7 +135,6 @@ contract DeployOPChainOutput {
     L1CrossDomainMessenger internal _l1CrossDomainMessengerProxy;
     OptimismPortal2 internal _optimismPortalProxy;
     DisputeGameFactory internal _disputeGameFactoryProxy;
-    DisputeGameFactory internal _disputeGameFactoryImpl;
```

Comment:
since the DGF is MCP ready, we can keep the already existing impl deployment in DeployImplementations